src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -1573,6 +1532,43 @@ protected virtual ChocolateyConfiguration set_package_config_for_upgrade(Chocola
             return originalConfig;
         }
 
+        private void validate_nuspec(string nuspecFilePath, ChocolateyConfiguration config)
+        {
+            var rules = _ruleService.validate_rules(nuspecFilePath);
+
+            if (!config.PackCommand.PackThrowOnUnsupportedElements)
+            {
+                rules = rules.Where(r => !r.Message.contains("not supported"));
+            }
+
+            var hasErrors = false;
+
+            foreach (var rule in rules)
+            {
+                switch (rule.Severity)
+                {
+                    case infrastructure.rules.RuleType.Error:
+                        this.Log().Error("ERROR: " + rule.Message);
+                        hasErrors = true;
+                        break;
+
+                    case infrastructure.rules.RuleType.Warning:
+                        this.Log().Warn("WARNING: " + rule.Message);
+                        break;
+
+                    case infrastructure.rules.RuleType.Information:
+                        this.Log().Info(rule.Message);

```

Comment:
Why not include an `INFO:` similar to `WARNING:` and `ERROR:`?

Reply:
I wasn't sure if we should call out informational rules explicitly.
I don't mind either way though, so I'll get it changed.